driver-core/src/test/java/com/datastax/driver/core/CCMBridgeTest.java

```diff
@@ -32,6 +32,7 @@ public class CCMBridgeTest extends CCMTestsSupport {
   @Test(groups = "short")
   @ScyllaVersion(
       maxOSS = "6.2.0",
+      maxEnterprise = "2024.1",
```

Comment:
This should be 2025.1, right?

Reply:
Yes, fixed in https://github.com/scylladb/java-driver/pull/493